src/python/pants/backend/jvm/tasks/coverage/jacoco.py

```diff
@@ -0,0 +1,73 @@
+# coding=utf-8
+# Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+import functools
+
+from pants.backend.jvm.tasks.coverage.engine import CoverageEngine
+from pants.subsystem.subsystem import Subsystem
+
+
+class Jacoco(CoverageEngine):
+  """Class to run coverage tests with Jacoco."""
+
+  class Factory(Subsystem):
+    options_scope = 'jacoco'
+
+    @classmethod
```

Comment:
I think that this should be an instance method: the idea is that you get an instance of the subsystem/Factory with, e.g. `global_instance()`, and then call the instance method (which can consume the options for that instance using `self.get_options()`, iirc) to create the `CoverageEngine`.